chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -346,6 +346,8 @@ public CertifiedProductController(final CertifiedProductUploadHandlerFactory upl
 
         certifiedProduct = validateCertifiedProduct(certifiedProduct);
```

Comment:
Why validate this product when the next line removes the warning/error messages?